docs/dev-guide/SERVICES.md

```diff
@@ -519,17 +519,17 @@ metadata:
   namespace: kube-system
 data:
   contiv.conf: |-
-    useNoOverlay: true
-    useSRv6Interconnect: true
+    nodeToNodeTransport: nooverlay
+    useSRv6ForServices: true
     ...
 ```
 `manifest.yaml` file is generated by Helm, so you can alternatively change the helm template 
 input values in [values.yaml][values.yaml] (or [values-arm64.yaml][values-arm64.yaml])
 ```
 ...
 contiv:
-  useNoOverlay: false
-  useSRv6Interconnect: false
+  nodeToNodeTransport: "nooverlay"
+  useSRv6ForServices: false
```

Comment:
shouldn't this be true?

Reply:
fixed and enhanced in https://github.com/contiv/vpp/pull/1560/commits/8d207a7c5cf9bb116955cef8f6a8be3c50b3494b